src/transaction/locator_sr.c

```diff
@@ -13700,3 +13701,130 @@ xlocator_demote_class_lock (THREAD_ENTRY * thread_p, const OID * class_oid, LOCK
 {
   return lock_demote_class_lock (thread_p, class_oid, lock, ex_lock);
 }
+
+
+// *INDENT-OFF*
+int 
+locator_multi_insert_force (THREAD_ENTRY * thread_p, HFID * hfid, OID * class_oid, std::vector<OID> oids,
+                            std::vector<RECDES> recdes, int has_index, int op_type, HEAP_SCANCACHE * scan_cache,
+                            int *force_count, int pruning_type, PRUNING_CONTEXT * pcontext,
+                            FUNC_PRED_UNPACK_INFO * func_preds, UPDATE_INPLACE_STYLE force_in_place)
+{
+  int error_code = NO_ERROR;
+  size_t accumulated_records_size = 0;
+  size_t heap_max_page_size;
+  std::vector <RECDES> recdes_array;
+  std::vector <OID> oids_array;
+  std::vector <VPID> heap_pages_array;
+
+  // Safeguard
+  assert (recdes.size() == oids.size ());
```

Comment:
Oid's should be generated by locator_insert_force. What is the meaning of input OID's?